components/apimgt/org.wso2.carbon.apimgt.impl/src/test/java/org/wso2/carbon/apimgt/impl/APIProviderImplTest.java

```diff
@@ -142,14 +142,7 @@
 import java.io.InputStream;
 import java.lang.reflect.Field;
 import java.lang.reflect.Modifier;
-import java.util.ArrayList;
-import java.util.HashMap;
-import java.util.HashSet;
-import java.util.List;
-import java.util.Map;
-import java.util.Set;
-import java.util.SortedMap;
-import java.util.TreeMap;
+import java.util.*;
```

Comment:
let's not use wildcard imports

Reply:
Fixed with https://github.com/wso2/carbon-apimgt/pull/10487/commits/f7cce7cd884bf9fc58efecaaad4359ffe282bc84